source/MonoGame.Extended.Input/MouseStateExtended.cs

```diff
@@ -17,8 +17,8 @@ public MouseStateExtended(MouseState currentMouseState, MouseState previousMouse
 
         public int X => _currentMouseState.X;
         public int Y => _currentMouseState.Y;
-        public Point Position => _currentMouseState.Position;
-        public bool PositionChanged => _currentMouseState.Position != _previousMouseState.Position;
+        public Point Position => new Point(_currentMouseState.X, _currentMouseState.Y);
```

Comment:
had to check, this is due to FNA not implementing a `MouseState.Position` property and only providing the `X` and `Y` properties.  Just adding this for notes, this change is fine.